samples/samples/snippets.py

```diff
@@ -969,6 +970,47 @@ def insert_singers(transaction):
     # [END spanner_dml_standard_insert]
 
 
+# [START spanner_get_commit_stats]
+def log_commit_stats(instance_id, database_id):
+    """Inserts sample data using DML and displays the commit statistics. """
+    # By default, commit statistics are logged via stdout at level Info.
+    # This sample uses a custom logger to access the commit statistics.
+    class CommitStatsSampleLogger(logging.Logger):
+
+        def __init__(self):
+            self._last_commit_stats = None
+            super().__init__("commit_stats_sample")
+
+        def info(self, msg, *args, **kwargs):
+            if kwargs['extra'] and 'commit_stats' in kwargs['extra']:
+                self._last_commit_stats = kwargs['extra']['commit_stats']
+            super().info(msg)
+
+    spanner_client = spanner.Client()
+    instance = spanner_client.instance(instance_id)
+    database = instance.database(database_id, logger=CommitStatsSampleLogger())
+    database.log_commit_stats = True
+
+    def insert_singers(transaction):
+        row_ct = transaction.execute_update(
+            "INSERT Singers (SingerId, FirstName, LastName) "
+            " VALUES (110, 'Virginia', 'Watson')"
+        )
+
+        print("{} record(s) inserted.".format(row_ct))
+
+    database.run_in_transaction(insert_singers)
+    commit_stats = database.logger._last_commit_stats
```

Comment:
Does `_last_commit_stats` need to be private?

Reply:
It does not. Updated :+1: